Mathlib/MeasureTheory/Measure/Lebesgue/EqHaar.lean

```diff
@@ -70,6 +70,10 @@ def TopologicalSpace.PositiveCompacts.piIcc01 (ι : Type*) [Fintype ι] :
       imp_true_iff, zero_lt_one]
 #align topological_space.positive_compacts.pi_Icc01 TopologicalSpace.PositiveCompacts.piIcc01
 
+theorem TopologicalSpace.PositiveCompacts.piIcc01_measurable (ι : Type*) [Fintype ι] :
+    MeasurableSet (TopologicalSpace.PositiveCompacts.piIcc01 ι : Set (ι → ℝ)) := by
+  exact MeasurableSet.pi countable_univ (fun _ _ => measurableSet_Icc)
```

Comment:
Why do you need this theorem? Can't you just use [IsCompact.measurableSet](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.html#IsCompact.measurableSet)?

Reply:
Oops, I guess the reason is that I didn't know about this result. It does make things a lot easier and this result is not needed anymore.